src/main/groovy/com/github/prokod/gradle/crossbuild/utils/CrossBuildPluginUtils.groovy

```diff
@@ -63,7 +63,13 @@ class CrossBuildPluginUtils {
         }
     }
 
+    //this feature is undocumented!
     static String qmarkReplace(String template, String replacement) {
-        template.replaceAll('\\?', replacement)
+        String newTag = template.replaceAll('\\?', replacement)
+        if(newTag == template) {
```

Comment:
I think this _? replacement should be either documented or removed.
imageine if some wnated to have the extsions _?.?? :roll_eyes: